app/views/HomeView.coffee

```diff
@@ -94,7 +94,9 @@ module.exports = class HomeView extends RootView
     else
       anchorText = anchor.text
     if anchorText
-      window.tracker?.trackEvent "Link: #{anchorText}", category: 'Homepage', ['Google Analytics']
+      @homePageEvent("Link: #{anchorText}", {}, ['Google Analytics'])
+    else
+      @homePageEvent("Link:", {clicked: e?.currentTarget?.host or "unknown"}, [])
```

Comment:
I assume this is a requirement?

Reply:
This is not a requirement. There is no analytics spec, just capture things in as much detail as required.
This extension captures links that aren't translated. Things like the social media buttons in the footer are now captured.
Therefore now we can observe how many users click to our Facebook or Twitter.